lib/module.js

```diff
@@ -460,29 +461,23 @@ Module.runMain = function() {
 };
 
 Module._initPaths = function() {
-  const isWindows = process.platform === 'win32';
+  modulePaths = [];
```

Comment:
I'd use `modulePaths.length = 0`.
That doesn't create a new object which has to be garbage collected